docs/changelog.rst

```diff
@@ -14,6 +14,11 @@ Fixed
 
 - Removed internal dependency on ``pip.utils``, which makes the library compatible with pip 10. `#70`_ (`butorov`_, `Stranger6667`_)
 
+Added
```

Comment:
This block refers to already published release, please put it to `Unreleased` section above.

Reply:
done